File: examples/expressions.c
#include "../runtime/pyrinas.h"

int main() {
    int x = 10;
    float y = 3.500000;
    float result_add = (x + y);
    int result_sub = (x - 5);
    float result_mul = (y * 2.000000);
    float result_div = (x / 4.000000);
    printf("%d\n", result_add);
    printf("%d\n", result_sub);
    printf("%d\n", result_mul);
    printf("%d\n", result_div);
    int is_equal = x == 10;
    int is_greater = y == x;
    printf("%d\n", is_equal);
    printf("%d\n", is_greater);
    return 0;
}

File: examples/expressions.py
def main() -> int:
    x: int = 10
    y: float = 3.5
    
    result_add: float = x + y
    result_sub: int = x - 5
    result_mul: float = y * 2.0
    result_div: float = x / 4.0
    
    print(result_add)
    print(result_sub)
    print(result_mul)
    print(result_div)
    
    is_equal: bool = x == 10
    is_greater: bool = y > x
    
    print(is_equal)
    print(is_greater)
    
    return 0